src/app/public/modules/file-attachment/file-attachment-label.component.html

```diff
@@ -0,0 +1,3 @@
+<span class="sky-control-label">
```

Comment:
Why `sky-control-label` instead of something more meaningful like `sky-file-attachment-label`? Is this a pattern I'm missing?

Reply:
Used this to get the same styling as other labels that are sharing the form that have `class=sky-control-label`. If we want to avoid using this class here I could style `sky-file-attachment-label` instead.